docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4154 @@
+{
```

Comment:
<p>Add a subheading here to flag up the switch to Bayesian methods</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>